Source/Core/AST/Program.cs

```diff
@@ -0,0 +1,587 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Microsoft.Boogie.GraphUtil;
+
+namespace Microsoft.Boogie;
+
+public class Program : Absy
```

Comment:
Moved out of `Absy.cs`. Updated one line to `rc.Options.OutputWriter.WriteLine("Warning: Ignoring implementation {0} because of translation resolution errors",`